src/index.js

```diff
@@ -1,52 +1,52 @@
 import React from "react"
 import PropTypes from "prop-types"
 
-function OutboundLink(props) {
+const NEWTAB = '_blank';
+const MIDDLE_CLICK = 1;
+
+function OutboundLink({ eventName, eventDetails, href, target, onClick, onMouseDown, children, ...rest }) {
+  const handleClick = e => {
+    const amplitudeEventName = eventName || window.amplitudeEventTypes.outboundLinkClick;
+    const amplitudeDetails = Object.assign({ href: href }, eventDetails)
+    const sameTarget = target !== NEWTAB;
+    const normalClick = !(e.ctrlKey || e.shiftKey || e.metaKey || e.button === MIDDLE_CLICK);
+
+    if (sameTarget && normalClick) {
+      e.preventDefault();
+      window.amplitude.getInstance().logEvent(amplitudeEventName, amplitudeDetails, () => {
+        window.location.href = href;
+      });
+    } else {
+      window.amplitude.getInstance().logEvent(amplitudeEventName, amplitudeDetails);
+    }
+
+    onClick && onClick(e);
```

Comment:
Please refactor these function conditions to use `if` condition blocks.

Reply:
sure! will do!